tests/simulation/service/test_services_v2.py

```diff
@@ -182,11 +177,14 @@ def test_init():
     assert data["operating_condition"]["alpha"]["value"] == 0
     assert data["operating_condition"]["alpha"]["units"] == "degree"
     assert "velocity_magnitude" not in data["operating_condition"].keys()
+    # to convert tuples to lists:
+    data = json.loads(json.dumps(data))
+    compare_dict_to_ref(data, "../../ref/simulation/service_init.json")
 
 
 def test_validate_init_data_errors():
 
     data = services.get_default_params(unit_system_name="SI", length_unit="m")
     _, errors, _ = services.validate_model(params_as_dict=data, unit_system_name="SI")
     json.dumps(errors)
-    assert len(errors) == 1
+    assert len(errors) == 3
```

Comment:
Can you explicitly have a check on the errors to make sure we know exactly what the errors are for default values?